comixed-frontend/src/app/comic-import/index.ts

```diff
@@ -30,14 +38,16 @@ interface RouterStateUrl {
 
 export interface AppState {
   router: fromRouter.RouterReducerState<RouterStateUrl>;
-  comicImport: ComicImportState;
+  [FIND_COMIC_FILES_FEATURE_KEY]: FindComicFilesState;
+  [SELECTED_COMIC_FILES_FEATURE_KEY]: SelectedComicFilesState;
 }
 
 export type State = AppState;
 
 export const reducers: ActionReducerMap<AppState> = {
   router: fromRouter.routerReducer,
-  comicImport: fromImporting.reducer
+  [FIND_COMIC_FILES_FEATURE_KEY]: fromFindComicFiles.reducer,
+  [SELECTED_COMIC_FILES_FEATURE_KEY]: fromSelectedComicFiles.reducer
 };
 
 export const metaReducers: MetaReducer<AppState>[] = !environment.production
```

Comment:
This conditional operation returns the same value whether the condition is "true" or "false" => [] 
Couldn't we simplify this?

Reply:
We could. But, like the empty SCSS files, I've left them in since we will take advantage of MetaReducers in future once the API is fully matured.